src/main.rs

```diff
@@ -37,6 +41,33 @@ mod lsp_data;
 #[cfg(test)]
 mod test;
 
+// FIXME doesn't seem to fit in existing modules
+struct VfsWrapper(Arc<Vfs>);
```

Comment:
Is the reason for having this just because the VFS is in an upstream crate, or is there some other constraint?